source/localizable/feedspec/index.en.html.md.erb

```diff
@@ -177,12 +177,14 @@ Name                                                   | Type    | Length
 [Shipping Costs](#shipping-costs)                      | Decimal | <span class="mono">-</span>    | No       | `3.22`
 [Color](#color)                                        | String  | <span class="mono">100</span>  | No<b><sup>3</sup></b>| `Black`
 [Description](#description)                            | String  | <span class="mono">1000</span> | No<b><sup>4</sup></b>| `SAMSUNG Galaxy A70 as a phablet features 6.7 inch display that affords you a vivid and different visual experience. 4 cameras, 32.0MP + 5.0MP + 8.0MP rear camera + 32.0MP front camera, you can enjoy images with 1080 x 2400 high resolution. It comes with most of the features we've come to expect from a phablet, including 6GB RAM 128GB ROM storage equipped with Android Snapdragon 675 and 4500mAh big capacity battery that you can play games faster.`
-[Quantity](#quantity)                                  | Integer  | <span class="mono">8</span>   | No | `10`
+[Quantity](#quantity)                                  | Integer | <span class="mono">8</span>    | No | `10`
+[Size Variations](#size-variation-attributes)          | Embedded XML Element | <span class="mono">-</span> | No<b><sup>5</sup></b> | `<variations>`<br />&nbsp;&nbsp;`<variation>...</variation>`<br />&nbsp;&nbsp;`<variation>...</variation>`<br />`</variations>`
 
 <small><b><sup>1</sup></b> Required by all fashion items with sizes, additional images</small><br />
 <small><b><sup>2</sup></b> Required by all shops that use weight in shipping cost rules</small><br />
 <small><b><sup>3</sup></b> Required by all fashion items</small><br />
-<small><b><sup>4</sup></b> Strongly recommended for all products</small>
+<small><b><sup>4</sup></b> Strongly recommended for all products</small><br />
+<small><b><sup>5</sup></b> Strongly recommended for all fashion items with sizes</small>
```

Comment:
I suggest to be more generic:
"Strongly recommended for all items that require sizes"